indexer/ftypes_matcher.cpp

```diff
@@ -481,6 +481,11 @@ IsPoiChecker::IsPoiChecker() : BaseChecker(1 /* level */)
     m_types.push_back(classif().GetTypeByPath({type}));
 }
 
+IsGeneralAmenityChecker::IsGeneralAmenityChecker() : BaseChecker(2 /* level */)
```

Comment:
Should be 1 /* level */ here.

Reply:
I want this checker to only match "amenity" while not matching subtypes like "amenity-bar" or "amenity-restaurant"... I've been trying different things and I've found that using level=2 I can get this behavior. If I change it to level=1 then this checker matches both "amenity" and all subtypes like "amenity-bar" or "amenity-restaurant"...

Reply:
Ah, I see, good point.